_sass/addon/syntax.scss

```diff
@@ -2,8 +2,8 @@
 * The syntax highlight.
 */
 
-@import 'colors/syntax-light';
-@import 'colors/syntax-dark';
+@import '../colors/syntax-light';
+@import '../colors/syntax-dark';
```

Comment:
Why change these two lines?

```suggestion
@import 'colors/syntax-light';
@import 'colors/syntax-dark';
```

Reply:
because this is the correct location for the file, using VS Code, it opens to it correctly, otherwise without `../` it does not.

Reply:
In Jekyll `@import` is searching for files starting from `sass_dir `, it doesn't matter if vscode can read the path or not.

Reply:
Hi @classicrocker883, are you going to undo the changes here?